docs/knowledgebase/getting-started/index.md

```diff
@@ -11,7 +11,7 @@ Unix-based operating systems.
 
 ## Unix-Based Operating Systems
 
-Substrate development is easiest on Unix-based operating systems like macOS or Linux. The examples
+Substrate development is optimized for Unix-based operating systems like macOS or Linux. The examples
```

Comment:
I'm uncomfortable with this terminology - it feels a little too strong to me. I don't think anyone has made any specific decisions to ensure that Substrate development is more performant on Unix-based systems, for instance, which is what I tend to think when the word "optimized" is used. From my POV, Substrate development just happens to be easiest on Unix-based systems because that's what most of the Substrate developers use.

Reply:
I prefer optimized as easiest denotes simply which Im not sure is the case. Also, not a hill I am willing to die on :)